bot/exts/backend/error_handler.py

```diff
@@ -97,13 +96,14 @@ async def on_command_error(self, ctx: Context, e: errors.CommandError) -> None:
             # MaxConcurrencyReached, ExtensionError
             await self.handle_unexpected_error(ctx, e)
 
-    @staticmethod
-    def get_help_command(ctx: Context) -> t.Coroutine:
+    async def send_command_help(self, ctx: Context) -> None:
         """Return a prepared `help` command invocation coroutine."""
         if ctx.command:
-            return ctx.send_help(ctx.command)
+            self.bot.help_command.context = ctx
```

Comment:
What caused this?

Reply:
When you call the help command directly, the context for the help cog gets updated, but here that doesn't happen anywhere and the context is empty.

I'm not at a computer at the moment, but you'll be able to see the bug that happens if you remove that line and then try to do `!clean &$#`